lib/spack/spack/test/cmd/clean.py

```diff
@@ -38,14 +38,14 @@ def __call__(self, *args, **kwargs):
     'mock_packages', 'config', 'mock_calls_for_clean'
 )
 @pytest.mark.parametrize('command_line,counters', [
-    ('mpileaks', [1, 0, 0, 0, 0]),
-    ('-s',       [0, 1, 0, 0, 0]),
-    ('-sd',      [0, 1, 1, 0, 0]),
-    ('-m',       [0, 0, 0, 1, 0]),
-    ('-f',       [0, 0, 0, 0, 1]),
-    ('-t',       [0, 0, 0, 0, 1]),
-    ('-a',       [0, 1, 1, 1, 1]),
-    ('',         [0, 0, 0, 0, 0]),
+    ('mpileaks', [1, 0, 0, 0, 0, 0]),
+    ('-s',       [0, 1, 0, 0, 0, 0]),
+    ('-sd',      [0, 1, 1, 0, 0, 0]),
+    ('-m',       [0, 0, 0, 1, 0, 0]),
+    ('-f',       [0, 0, 0, 0, 1, 0]),
+    ('-t',       [0, 0, 0, 0, 0, 1]),
+    ('-a',       [0, 1, 1, 1, 1, 1]),
+    ('',         [0, 0, 0, 0, 0, 0]),
```

Comment:
counters should really be a dict with informatively named keys.

Reply:
I agree, but it's non-trivial to refactor and I think it's outside the scope of this PR.